src/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/3b9cd09213.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>To do List</title>
+  </head>
+  <body>
+    <main>
+      <h1>To do List</h1>
+      <div class="container">
+        <div class="row title ">
+            <h2>today's To Do</h2>
+            <i class="fa-solid fa-list-check"></i>
+        </div>
+        <hr>
+        <input type="text" id="newtask" placeholder="Add a new task...." />
+        <ul class="list-container"></ul>
+        <h3 class="deleteCompleteTask">Clear all Complete task</h3>

```

Comment:
By convention, CSS class names use the `kebab-case`. It's important to follow conventions because it reduces the effort needed to read and understand code. So, you could change `deleteCompletedTask` to `delete-completed-task`.

Apart from that, I really admire how you implemented everything. Kudos to you 🥂

Reply:
Ok Thank @kwesiObuobi